lib/fluent/plugin/bigquery/writer.rb

```diff
@@ -37,6 +37,7 @@ def create_table(project, dataset, table_id, record_schema)
           if @options[:time_partitioning_type]
             definition[:time_partitioning] = {
               type: @options[:time_partitioning_type].to_s.upcase,
+              field: @options[:time_partitioning_field].to_s,
```

Comment:
Unless @options[:time_partitioning_field] exists, field should be nil.
I think so.
Could you fix it?

Reply:
Yep, you're right. I've just fixed it.